docs/_posts/2018-01-15-jekyll-3-7-1-released.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Jekyll 3.7.1 Released"
+date: 2018-01-15 11:21:00 +0530
+author: ashmaroli
+version: 3.7.1
+categories: [release]
+---
+
+Close on the heels of shipping 3.7.0, we were informed of a couple of
+regressions due to the changes made in that release. In due time, Team Jekyll
+set out to address those issues as early as possible.
+
+Days later, here we're announcing 3.7.1 that aims to set things in order. The
+highlights being:
+
+  * A major regression in 3.7.0 was that when a Front Matter Default was
+    configured with a `scope["path"]` set to a directory, Jekyll would scan
+    that directory for any subfolders and files, for each document in that
+    `path`.
+    Though this is intended, it increases build times in proportion to the size
+    of the said directory. <br />
+    We addressed this by having Jekyll scan the directory path only if the user
+    explicitly configures the `scope["path"]` using wildcards. <br />
+    Read our documentation for more details. <br />
```

Comment:
Link ?

Reply:
Working on a document to attach to the associated PR.. will update after it gets merged..

Reply:
Are all the `<br />` tags necessary? Can't we use Markdown line breaks?

Reply:
Markdown (or GFM, not sure..) converts `\n\s*` into `\s` and `\n\n` into `\n<p>`
I wanted to just insert simple breaks.. I'll locally "preview" the document once more and adopt the better outcome..

Reply:
Link to the docs is still missing here.